crates/viewer/re_space_view_map/src/map_space_view.rs

```diff
@@ -214,20 +216,59 @@ Displays geospatial primitives on a map.
             Err(err) => return Err(err),
         };
         egui::Frame::default().show(ui, |ui| {
+            let mut picked_instance = None;
+
             let some_tiles_manager: Option<&mut dyn Tiles> = Some(tiles);
-            let map_widget = ui.add(
-                Map::new(some_tiles_manager, map_memory, default_center_position)
-                    .with_plugin(geo_points_visualizer.plugin()),
+            let map_response = ui.add(
+                Map::new(some_tiles_manager, map_memory, default_center_position).with_plugin(
+                    geo_points_visualizer.plugin(ctx, query.space_view_id, &mut picked_instance),
+                ),
             );
 
-            if map_widget.double_clicked() {
+            if let Some(picked_instance) = picked_instance {
+                map_response.clone().on_hover_ui_at_pointer(|ui| {
```

Comment:
this is slightly more idiomatic, in case `on_hover_ui_at_pointer` mutates `Self`

```suggestion
                map_response = map_response.on_hover_ui_at_pointer(|ui| {
```

Reply:
ofc! 🤦🏻  That clone definitely felt off (but it was late)